src/app/modules/training-record/services/record.service.spec.ts

```diff
@@ -79,4 +89,31 @@ describe('RecordService', () => {
     req.flush({});
   });
 
+  it('should get records', () => {
+    const service: RecordService = TestBed.get(RecordService);
+    const offset = 0;
+    const limit = 20;
+
+    service.getRecords(offset, limit).subscribe(
+      res => { expect(res.results.length).toEqual(2); });
+
+    const url = `${environment.API_URL}/records/?offset=${offset}&limit=${limit}`;
+
+    const req = httpTestingController.expectOne(url);
+    expect(req.request.method).toEqual('GET');
+    req.flush({count: 2, results: [{}, {}]});
+  });
+
+  it('should ignore errors during getting records', fakeAsync(() => {
```

Comment:
What are you exactly testing in this unit test case? I don't see any `expect` statements. And why are you using `fakeAsync`?